addons/mrp_subcontracting/wizard/mrp_product_produce.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+class MrpProductProduce(models.TransientModel):
+    _inherit = 'mrp.product.produce'
+
+    subcontract_move_id = fields.Many2one('stock.move')
+
+    def continue_production(self):
+        action = super(MrpProductProduce, self).continue_production()
+        action['context'] = dict(action['context'], default_subcontract_move_id=self.subcontract_move_id.id)
+        return action
+
+    def _generate_produce_lines(self):
```

Comment:
add a comment
When the user did not have to use the produce wizard, this method is used to appropriately set the lot_produced_id and appropriately create a raw stock move line.